crates/turborepo-lib/src/run/summary/spaces.rs

```diff
@@ -40,6 +47,23 @@ pub struct SpacesClientHandle {
     tx: Sender<SpaceRequest>,
 }
 
+/// A spaces client with functionality limited to sending task information
+/// This client should only live while processing a task
+pub struct SpacesTaskClient {
```

Comment:
nit: I'm not convinced this needs to be a separate type versus a type alias

Reply:
I'm not going to die on this hill, but having this as it's own type allows callers to not worry about the preamble of converting task information into a summary and handling error coercion. Also it doesn't leak implementation details of how one sends a task.